core/infra/server/http_server_with_cappuccino.cpp

```diff
@@ -82,6 +119,22 @@ namespace http {
             auto data = request->json();
             std::string uuid;
 
+            Api::Domain domain;
+            domain.set_ownerpublickey("pubkey1");
+            domain.set_name("name");
+            auto txDomain = TransactionBuilder<Remove<Domain>>()
+                .setSenderPublicKey("karin")
+                .setDomain(domain)
+                .build();
+
+            Torii(
+                Sumeragi::NewStub(grpc::CreateChannel(
+                    config::PeerServiceConfig::getInstance().getMyIp() + ":50051",
```

Comment:
Hardcoded port. As I remember, config manager has option to get grpc port from config file.